installer/scripts/genRubyInstaller.sh

```diff
@@ -52,14 +52,20 @@ if [ -z "$SOURCE_DIR" -o ! -d $SOURCE_DIR ]; then
     exit 1
```

Comment:
I'm after better diagnostics here (which is what I had done in prior reviews).

Imagine an error like "This script must be run from the OMS build directory" rather than "Source directory does not exist".

One tells you just what to do, another requires that you look at a script and diagnose the problem.

I'm after two basic checks (which I sent in a prior review).

1. Are we starting off in the correct directory? If not, we'll fail,
2. Are we doing a universal build, and has the build been done? If not, we'll fail.

I'd like those to be checked for since those are both very common mistakes.